apiserver/facades/client/application/service.go

```diff
@@ -96,6 +97,10 @@ type ApplicationService interface {
 	// satisfying [applicationerrors.UnitNotFoundError] if the unit is not found.
 	GetUnitLife(ctx context.Context, name string) (life.Value, error)
 
+	// GetUnitUUID returns the UUID for the named unit, returning an error
+	// satisfying [applicationerrors.UnitNotFound] if the unit doesn't exist.
```

Comment:
**todo:** applicationerrors is not aliased in import in this file: you need to use the fully qualified name

```suggestion
	// satisfying [github.com/juju/juju/domain/application/errors.UnitNotFound] if the unit doesn't exist.
```

(please fix other occurence in this file also, but check if there are accurate. For instance, `DeleteApplication` usage doesn't rely on UnitNotFound in the current context. This error doesn't need to be documented in this interface.)

Reply:
On it!

Reply:
I think I will just remove references to the error type here